app/script/conversation/ConversationService.coffee

```diff
@@ -381,12 +381,15 @@ class z.conversation.ConversationService
   @param payload [Object] Payload to be posted
   @option [OtrRecipients] recipients Map with per-recipient data
   @option [String] sender Client ID of the sender
-  @param force_sending [Boolean] Should the backend ignore missing clients
+  @param report_missing [Array<String>|Boolean] Option to handle missing clients
   @return [Promise] Promise that resolve when the message was sent
```

Comment:
Mixing up these two values is a bit misleading. Since `report_missing=true` actually forces the message to be send instead of reporting missing clients.